Product/Production/Services/PolicyEngineCore/src/main/java/gov/hhs/fha/nhinc/policyengine/adapter/pip/XACMLCreator.java

```diff
@@ -360,11 +329,10 @@ else if ((sHL7DateTime != null) && (sHL7DateTime.length() > 8)) {
                 sXMLDate = oXMLDateTimeFormatter.format(oHL7Date);
             } catch (Exception e) {
                 String sErrorMessage = "Date-time had invalid HL7 format: " + sHL7DateTime + ".  Error = "
-                        + e.getMessage();
+                    + e.getMessage();
                 throw new AdapterPIPException(sErrorMessage, e);
             }
-        }
-        // Do not know format - put it in as is.
+        } // Do not know format - put it in as is.
```

Comment:
Making the following comment as an FYI for future reference, DON'T make another commit to change this:

I know you made this change to stay consistent with the rest of the class, but it's actually the rest of the class that's wrong -- comments should always be on their own lines.  In this case, I would have suggested moving "// Do not know format..." into the else, right about the "sXMLDate" assignment.  That way, the comment gets its own line, it's unambiguous where and why the comment belongs, and as an afterthought (but still an added bonus) -- it helps our CheckStyle finding count :)